src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXOverrideMismatch/PXOverrideMethodSymbolComparer.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using System.Linq;
+using Acuminator.Utilities.Common;
+
+namespace Acuminator.Analyzers.StaticAnalysis.PXOverrideMismatch
+{
+	public class PXOverrideMethodSymbolComparer : IEqualityComparer<IMethodSymbol>
+	{
+		public static PXOverrideMethodSymbolComparer Instance = new PXOverrideMethodSymbolComparer();
+
+		public bool Equals(IMethodSymbol src, IMethodSymbol ext)
```

Comment:
please use full names, no shortcuts. I would propose `sourceMethod` and `pxOverrideMethod`.